src/FsAutoComplete.Core/Commands.fs

```diff
@@ -212,8 +213,9 @@ type Commands<'analyzer> (serialize : Serializer, backgroundServiceEnabled) =
         decl.NamespaceToOpen
         |> Option.bind (fun n ->
             state.CurrentAST
-            |> Option.map (fun ast -> ParsedInput.findNearestPointToInsertOpenDeclaration (pos.Line) ast idents TopLevel )
-            |> Option.map (fun ic -> n, ic.Pos.Line, ic.Pos.Column, ic.ScopeKind.ToString()))
+            |> Option.map (fun ast -> ParsedInput.findNearestPointToInsertOpenDeclaration (pos.Line) ast idents Nearest)
```

Comment:
changed the insert position for the open namespace to `Nearest` from `TopLevel`, because in my experience folks want opens scoped to the containing module and not the top-level module.